src/lib/computeDiskPercent.ts

```diff
@@ -0,0 +1,11 @@
+import memoize from 'lodash/memoize'
+
+const compute = (usage: number | string, quota: number | string): string => {
+  const parsedUsage = parseInt(`${usage}`, 10)
+  const parsedQuota = parseInt(`${quota}`, 10) || 100000000000
```

Comment:
I'm wondering if we should just skip the calculus if no quota is found and display nothing instead of a wrong quota 🤔

Reply:
I checked it out, undefined quota means unlimited quota. Not sure what to do then. (it should happen on localhost only, but not sure)